packages/shared/src/components/RichTextEditor/plugins/LinkPlugin/index.ts

```diff
@@ -7,8 +7,11 @@ import { CustomFloatingLink } from './FloatingLink/CustomFloatingLink';
 import { LINK_PLUGIN } from './id';
 import { LinkButton } from './LinkButton';
 import { LinkMarkupElement } from './LinkMarkupElement';
+import { relativeUrlRegex, telOrMailRegex, urlRegexWithHttps } from './utils';
 
-export const isValidUrl = (url: string): boolean => !!url.match(/^((https?:\/\/|mailto:|tel:).+)|\/r\/.+/);
+export const isValidUrl = (url: string): boolean => {
```

Comment:
Also that function is worth to have some unit tests

Reply:
Thanks, done!